<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="backgroundWid" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#backgroundWid{
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(33, 134, 185, 255), stop:1 rgba(181, 77, 208, 255));}
</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>391</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 48pt &quot;Avenir Next&quot;;color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>Online Bookstore</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>270</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 24pt &quot;Didot&quot;; color: rgb(255, 255, 255);
background-color: rgb(71, 156, 194);</string>
     </property>
     <property name="text">
      <string> User Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signUpButton">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>340</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 24pt &quot;Didot&quot;; color: rgb(255, 255, 255);
background-color: rgb(71, 156, 194);</string>
     </property>
     <property name="text">
      <string>here!</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>340</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(255, 255, 255);
font: 24pt &quot;Didot&quot;;</string>
     </property>
     <property name="text">
      <string>Don't have an account? Sign up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AloginButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>270</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 24pt &quot;Didot&quot;; color: rgb(255, 255, 255);
background-color: rgb(71, 156, 194);</string>
     </property>
     <property name="text">
      <string>Admin Login</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
